cleanlab/multiannotator.py

```diff
@@ -0,0 +1,436 @@
+import numpy as np
+import pandas as pd
+from typing import List, Union, Tuple
+from cleanlab.rank import get_label_quality_scores
+from cleanlab.dataset import overall_label_health_score, _get_worst_class
+import warnings
+
+# TODO: unsure where to place this function? should it go to cleanlab.rank or stay here
```

Comment:
I'd add a one-line docstring and then move this method to: internal/label_quality_utils.py


https://github.com/cleanlab/cleanlab/blob/master/cleanlab/internal/label_quality_utils.py